wiki/Staff_Log/ru.md

```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - зачисление
```

Comment:
shouldnt have the English tags as well?

Reply:
should it? I've thought tags are always translation-specific, according to ASC

Reply:
it depends if you want to people using the Russian locale on web to come to RU or EN when searching using English words, since the RU/EN search results will be mixed together. but in practice the inclusion of English tags usually doesn't change anything, because even without English tags, the search boost on your own locale is super high and trumps over every kind of match for EN (e.g. searching "Developers" with Chinese locale has the ZH page 1st and EN page 3rd, even though that word appears nowhere in ZH and is the exact title of EN)

imo just follow ASC, there shouldn't be any problems with it, and it would be an issue with osu-web if there are. probably due for a checkup on wiki searches anyway bc the behavior has been weird for a long time. looking back I'm pretty sure I added English tags to a lot of translations without thinking this through 🤔